homeassistant/components/html5/notify.py

```diff
@@ -61,8 +58,8 @@ def gcm_api_deprecated(value):
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(ATTR_GCM_SENDER_ID): vol.All(cv.string, gcm_api_deprecated),
-        vol.Optional(ATTR_GCM_API_KEY): cv.string,
+        vol.Optional("gcm_sender_id"): vol.All(cv.string, gcm_api_deprecated),
+        vol.Optional("gcm_api_key"): cv.string,
         vol.Optional(ATTR_VAPID_PUB_KEY): cv.string,
         vol.Optional(ATTR_VAPID_PRV_KEY): cv.string,
         vol.Optional(ATTR_VAPID_EMAIL): cv.string,
```

Comment:
The vapid items should be required I think.

Reply:
> Changes to existing YAML configuration for these same existing integrations, will no longer be accepted.

I thought this was forbidden by [ADR-0010](https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision) but it is indeed better to make it required.

Fixed in: https://github.com/home-assistant/core/pull/83229/commits/02e78b3a743a10d46c3ea93db7d2e412b99d101f

Reply:
Good! We can now use `dict[key]` on lines 173-175 instead of `dict.get(key)`.